### ISSUE TITLE
can't install catboost with pip on docker instance

### ISSUE THREAD

**Issue by tipanverella** [catboost/catboost#2330]
Problem: ERROR on `pip install`
catboost version: 1.1.1
Operating System: Linux node1 4.4.0-210-generic #242-Ubuntu SMP
CPU: Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
GPU: (I don't know)

I ran the following on a docker play instance.

```bash
[node1] (local) root@192.168.0.18 ~
$ which python3
/usr/bin/python3
[node1] (local) root@192.168.0.18 ~
$ `which python3` --version
Python 3.10.4
[node1] (local) root@192.168.0.18 ~
$ pip install catboost
ERROR: Could not find a version that satisfies the requirement catboost (from versions: none)
ERROR: No matching distribution found for catboost
```

**Comment by dimbleby** [comment]
while that comment represents a real bug that y'all should want to fix, I don't think it has anything at all to do with this issue

**Comment by andrey-khropov** [comment]
> Problem: ERROR on `pip install` catboost version: 1.1.1 Operating System: Linux node1 4.4.0-210-generic #242-Ubuntu SMP CPU: Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz GPU: (I don't know)
> 
> I ran the following on a docker play instance.
> 
> ```shell
> [node1] (local) root@192.168.0.18 ~
> $ which python3
> /usr/bin/python3
> [node1] (local) root@192.168.0.18 ~
> $ `which python3` --version
> Python 3.10.4
> [node1] (local) root@192.168.0.18 ~
> $ pip install catboost
> ERROR: Could not find a version that satisfies the requirement catboost (from versions: none)
> ERROR: No matching distribution found for catboost
> ```

What does `which pip` output?

What about `python3 -m pip install catboost` ?

**Comment by nikdata** [comment]
I was creating a remote container using the base image 3.10-slim-bullseye. I tried installing cat boost and received the error messages that folks outlined above.

When I run `which pip`, I get the following output:

```
/usr/local/bin/pip
```

When I run `python -m pip install cat boost`, I get the following:

```
ERROR: Could not find a version that satisfies the requirement catboost (from versions: none)
ERROR: No matching distribution found for catboost

[notice] A new release of pip available: 22.3.1 -> 23.1.1
[notice] To update, run: pip install --upgrade pip
```

Here's a screenshot:

<img width="678" alt="image" src="https://user-images.githubusercontent.com/24734983/234152768-de6a63a9-b7b5-4264-a795-9fea5ea02d7d.png">

Reference 
- [GitHub Repo](https://github.com/nikdata/spaceship-py)

I hope this helps. Thanks!

**Comment by andrey-khropov** [comment]
One person in the Telegram chat mentioned that he was able to overcome the similar issue by specifying `—platform=linux/amd64` flag to pip.

**Comment by andrey-khropov** [comment]
> I was creating a remote container using the base image 3.10-slim-bullseye. I tried installing cat boost and received the error messages that folks outlined above.
> 
> When I run `which pip`, I get the following output:
> 
> ```
> /usr/local/bin/pip
> ```
> 
> When I run `python -m pip install cat boost`, I get the following:
> 
> ```
> ERROR: Could not find a version that satisfies the requirement catboost (from versions: none)
> ERROR: No matching distribution found for catboost
> 
> [notice] A new release of pip available: 22.3.1 -> 23.1.1
> [notice] To update, run: pip install --upgrade pip
> ```
> 
> Here's a screenshot:
> 
> <img alt="image" width="678" src="https://user-images.githubusercontent.com/24734983/234152768-de6a63a9-b7b5-4264-a795-9fea5ea02d7d.png">
> 
> Reference
> 
>     * [GitHub Repo](https://github.com/nikdata/spaceship-py)
> 
>     * Apple Silicon (M2)
> 
> 
> I hope this helps. Thanks!
> 
> **EDIT**: I think the following [issue](https://github.com/catboost/catboost/issues/2213#issuecomment-1515000094) fixes it (from a dev standpoint). Using the dev version, I was able to get the docker container up & running.

The most likely issue is that when you run on M1 Mac your platform will be `linux/arm64/v8`. CatBoost did not have packages supporting Linux on aarch64 on PyPI before release 1.2 (although it had been possible to build one yourself),  since release 1.2 it should work fine.
